examples/coagents-starter/agent-js/src/agent.ts

```
@@ -52,15 +51,15 @@ async function chat_node(state: AgentState, config: RunnableConfig) {
   //     the model to call tools that are defined in CopilotKit by the frontend.
   const modelWithTools = model.bindTools!(
     [
-      ...convertActionsToDynamicStructuredTools(state.copilotkit?.actions || []),
+      ...state.copilotkit?.actions,
```

Biome (2.1.2)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)